integration-tests/api/medusa-config.js

```diff
@@ -5,6 +5,7 @@ const DB_NAME = process.env.DB_TEMP_NAME
 
 const redisUrl = process.env.REDIS_URL || "redis://localhost:6379"
 const cacheTTL = process.env.CACHE_TTL || 15
+const enableResponseCompression = process.env.ENABLE_RESPONSE_COMPRESSION || true
```

Comment:
question: Do we want to default to a compressed response? Doesn't it require a specific set of headers sent from the client for the decompression to happen automatically? There's likely a trivial answer to this question, but I am not very familiar with the concept.

Reply:
No the package use the classic compression and just allow to set a level to the compression. It works out of the box

Reply:
Got it, thx. Just want to be sure that this does not affect existing users' setups.

Reply:
I guess that depends on how you want to approach it - whitelist requests or blacklist requests. Since I can't think of any endpoints that requires no compression, I defaulted to blacklist requests. The client still has an option to pass in `x-no-compression` to not compress it.